lib/pure/collections/lists.nim

```diff
@@ -983,6 +983,16 @@ func toSinglyLinkedList*[T](elems: openArray[T]): SinglyLinkedList[T] {.since: (
   for elem in elems.items:
     result.add(elem)
 
+func toSinglyLinkedRing*[T](elems: openArray[T]): SinglyLinkedRing[T] {.since: (2, 0, 0).} =
+  ## Creates a new `SinglyLinkedRing` from the members of `elems`.
+  runnableExamples:
+    from std/sequtils import toSeq
+    let a = [1, 2, 3, 4, 5].toSinglyLinkedRing
+    assert a.toSeq == [1, 2, 3, 4, 5]
+
+  for elem in elems.items:
```

Comment:
Shouldn't this call `initSinglyLinkedRing`?

Reply:
I don't believe so; according to: https://github.com/nim-lang/Nim/blob/cd84cd45ea5bc1de67dd28dab906ad6c16e7b092/lib/pure/collections/lists.nim#L113-L114
Also, the implementation is `discard`.

If there is some subtle reason or stylistic preference, I'm happy to update accordingly.

Reply:
Use `initSinglyLinkedRing` to have the code future-proof.

Reply:
Ack. I'll update to use `initSinglyLinkedRing`.